bokeh/properties.py

```diff
@@ -319,6 +322,10 @@ def has_ref(self):
         """ True if the property can refer to another HasProps instance."""
         raise NotImplementedError("Implement has_ref()")
 
+    def possibly_rethemed(self, obj, old):
```

Comment:
this was supposed to be renamed trigger_if_rethemed as it was in the subclass below